backend/ohq/management/commands/user_stat.py

```diff
@@ -0,0 +1,67 @@
+from django.core.management.base import BaseCommand
+from django.utils import timezone
+from datetime import timedelta
+import logging
+
+from ohq.models import Profile, Course, Question
+from ohq.statistics import (
+    user_calculate_questions_asked,
+    user_calculate_questions_answered,
+    user_calculate_time_helped,
+    user_calculate_time_helping,
+    user_calculate_students_helped,
+)
+
+logging.basicConfig(level=logging.DEBUG)
+logger = logging.getLogger(__name__)
+
+
+class Command(BaseCommand):
+    def add_arguments(self, parser):
+        parser.add_argument("--hist", action="store_true", help="Calculate all historic statistics")
+
+    def calculate_statistics(self, profiles, courses, earliest_date):
```

Comment:
The `earliest_date` parameter here is used to compute a `date` field, which I don't think is quite necessary. I think we can directly just use the `earliest_date` field wherever `date` is used.